realsense2_camera/src/fix_set_auto_exposure.cpp

```diff
@@ -145,8 +147,18 @@ static bool reacquire_device(rs2::context& context, const std::string& serial, r
   return false;
 }
 
-bool fixSetExposure(rs2::context& context, rs2::device &device, ros::Duration reset_wait_duration,
-                        int max_resets, ros::Duration fail_wait_duration)
+/**
+ * @brief Attempt to fix auto exposure setting by reset hardware and checking the actual configuration on devices.
+ * @param context The context used to query devices.
+ * @param serial The serial of the device to find.
+ * @param reset_wait_duration Time to wait after reset for the device to come back online.
+ * @param max_resets The maximum number of times to try resetting a device.
+ * @param fail_wait_duration Time to wait before another attempt to try fixing auto exposure setting.
+ * @param fail_wait_duration Auto exposure setting value to be set on the device. Enabled if true.
```

Comment:
typo, second `fail_wait_duration` should be `exposure`

Reply:
Oops. Will fix that thanks!